test/source/TestAlgorithm.cpp

```diff
@@ -2339,6 +2341,48 @@ int TestAlgorithm()
 		}
 	}
 
+	// issue #92
+	// disable in MSVC2013 because eastl::is_copy_constructible is always true...
```

Comment:
I gave up on trying to work around this bug and just disabled the test for MSVC2013.